.github/copilot-instructions.md

```diff
@@ -0,0 +1,121 @@
+# DLaB.Xrm - GitHub Copilot Agent Instructions
+
+This file provides essential information for GitHub Copilot agents working on the DLaB.Xrm repository.
+
+## Repository Overview
+
+**DLaB.Xrm** is a comprehensive .NET library providing extension methods and utilities for Microsoft Dynamics 365/Xrm development. The library includes:
+- Plugin development base classes and utilities
+- Extensive extension methods for IOrganizationService
+- Query expression helpers and SQL generation
+- IoC/Dependency Injection framework for plugins  
+- Workflow activity base classes
+- Test frameworks for unit testing Xrm components
+
+**Key Statistics:**
+- ~1,600 C# files across 7 projects
+- Multi-target framework: .NET Framework 4.6.2, 4.7.2, 4.8, and .NET 8.0
+- Uses MSTest for testing with 111+ test cases
+- Distributed as NuGet source packages (not compiled binaries)
```

Comment:
This project is distributed as both Nuget source packages (not compiled binaries) and normal Nuget packages (dlls), due to older versions of Microsoft. Dataverse not allowing the referencing of additional assemblies, and the newer version allowing it.    Can you add that it is distributed as both?